test/server/runner-test.js

```diff
@@ -248,6 +313,53 @@ describe('Runner', function () {
                 })
                 .catch(done);
         });
+
+        it('Should not create a new local browser connection if sources are empty', function (done) {
+            var startConnectionID = testCafe.createBrowserConnection().id;
```

Comment:
`ID` shouldn't be all upper case: `Id`. It's more likely `firstConnectionId`, BTW @VasilyStrelyaev check test names, please